package.json

```diff
@@ -19,7 +19,6 @@
     "prepare": "npm run build",
     "run": "npm run build && node ./dist/index.js",
     "test": "jest",
-    "test:cov": "npm run test -- --coverage",
```

Comment:
this is not needed as coverage is enabled int he jest config and runs with just `npm test`

Reply:
yes it is working because `collectCoverage` is set to true

Reply:
yes right